roles/openshift_logging/README.md

```diff
@@ -80,6 +80,12 @@ When both `openshift_logging_install_logging` and `openshift_logging_upgrade_log
 - `openshift_logging_es_number_of_shards`: The number of primary shards for every new index created in ES. Defaults to '1'.
 - `openshift_logging_es_number_of_replicas`: The number of replica shards per primary shard for every new index. Defaults to '0'.
 
+- `openshift_logging_eventrouter_sink`: Select a sink for eventrouter, supported 'stdout' and 'glog'. Defaults to 'stdout'.
```

Comment:
Lets move these entries to the role README

Reply:
Given neither role has its own README, I think it would be better to keep this PR only related to the introduction of eventrouter and then create separate PR which moves relevant README parts to appropriate roles